docs/conf.py

```diff
@@ -59,11 +59,18 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
-# The short X.Y version.
-version = '2.5'
+
+# Read the listed version
+with open("../pybind11/_version.py") as f:
+    code = compile(f.read(), "../pybind11/_version.py", "exec")
+loc = {}
+exec(code, loc)
+
 # The full version, including alpha/beta/rc tags.
-release = '2.5.dev1'
+release = loc["__version__"]
+
+# The short X.Y version.
+version = ".".join(release.split(".")[:2])
```

Comment:
If we can assume to always have two periods in the `release` string, then you can simplify this to `release.rsplit('.', 1)[0]`. Not sure if that's a good idea.

Reply:
I'm not really sure if we should make these different anymore, now that we use PEP 440 naming. I'll check, but I know that this shows up in the top left corner, and having the real version there would likely be better, IMO.

Reply:
(And we can always promise this will be at least length 3, so will always have two periods)

Reply:
> (And we can always promise this will be at least length 3, so will always have two periods)

Notice that I used `rstrip()` above, so anything other than length 3 wouldn't work.

Reply:
No, I didn't, so yes, it is allowed to be length 4. `2.6.0.dev1` is the current value, actually.

Reply:
In that case, using `split('.', 2)` would avoid splitting on the 3rd period, but as it's currently written is simpler.

Reply:
Got permission from @wjakob to drop the separation and just use `version`.